src/docs/pages/component-docs/shared/scaffold/scaffold.component.scss

```diff
@@ -11,36 +11,44 @@
         width: 100%;
         display: flex;
         justify-content: center;
+        .mat-tab-label {
+            // height: 50px;
+        }
```

Comment:
What's this doing... or not doing?

Reply:
There is still some style cleanup required as part of the overall scaffold - making the bottom Divider of the first NavItem align with the divider under the mat-tab-group was intensional.  I had to tweak the height of the tab group for this stuff to align, but it's still not perfect on mobile devices.